File: src/main/java/me/laus/tallinn/transport/client/external/TallinnExternalApiClient.java
package me.laus.tallinn.transport.client.external;

import me.laus.tallinn.transport.model.request.ExternalApiRequestScheme;

public abstract class TallinnExternalApiClient extends ExternalApiClient {
    private static final String TRANSPORT_TALLINN_URL = "transport.tallinn.ee";

    public TallinnExternalApiClient(String path) {
        super(TRANSPORT_TALLINN_URL, path);
    }
}

File: src/main/java/me/laus/tallinn/transport/client/external/TallinnExternalApiClient.kt
package me.laus.tallinn.transport.client.external

abstract class TallinnExternalApiClient(path: String) : ExternalApiClient(TRANSPORT_TALLINN_URL, path) {
    companion object {
        private const val TRANSPORT_TALLINN_URL = "transport.tallinn.ee"
    }
}